os/arch/arm/src/armv7-m/up_mpu.c

```diff
@@ -357,11 +356,24 @@ void mpu_get_register_value(uint32_t *regs, uint32_t region, uintptr_t base, siz
 		regs[0] = region;
 		regs[1] = (base & MPU_RBAR_ADDR_MASK) | region;
 		regs[2] = regval;
-	} else {
-#ifdef CONFIG_SUPPORT_COMMON_BINARY
-		putreg32(region, MPU_RNR);
-		putreg32((base & MPU_RBAR_ADDR_MASK) | region, MPU_RBAR);
-		putreg32(regval, MPU_RASR);
-#endif
+	}
+}
+
+/****************************************************************************
+ * Name: up_mpu_set_register
+ *
+ * Description:
+ *   Set MPU register values to real mpu h/w
+ *
+ ****************************************************************************/
+void up_mpu_set_register(uint32_t *mpu_regs)
```

Comment:
Can you pls rename this as mpu_set_register_value to make it similar to mpu_get_register_value?

Reply:
How about renaming `mpu_ger_register_value` to other? Because it looks reading the register to get value. But actually it is making (or combining) the value for mpu register.

Reply:
mpu_get_register_value and mpu_get_register_value are not similar as one changes the h/w settings and other is software manipulation..
we can change mpu_get_register_value to mpu_get_config_value or mpu_get_register_config_value

Reply:
Changed `mpu_get_register_value` to `mpu_get_register_config_value` as Sangam has suggested.